popSimManu.tex

```diff
@@ -835,6 +835,16 @@ \subsection*{Model quality control}
 Further details of our QC process can be found in our developer documentation
 (\url{https://stdpopsim.readthedocs.io/en/latest/development.html#}).
 
+The possibility for error and the importance of careful qualty control
+was inadvertantly illustrated during our own development process:
+during the final revisions of this paper, we noticed that the 
+OutOfAfrica\_3G09 model \citep{gutenkunst2009inferring}
+had not in fact gone through the QC process.
+Then, the QC process revealed that not only was our implementation slightly wrong --
+migration rates had not been set to zero to the European population in the longest-ago time period
+when there should have been only a African population.
```

Comment:
an African

Reply:
I would say "when there should have only been a single ancestral population" instead of "an African population". At 220kya, I don't think it makes sense to distinguish African vs non-African, even though we re-use that population index in the model.